.gitignore

```
@@ -8,6 +8,9 @@
 # Crash log files
 crash.log
 
+# Keep user settings local
+.vscode/*
```

```suggestion
.vscode/
```

Tiny thing . . With a `*` you say, "keep the empty .vscode folder but ignore its contents", whereas without it, it will ignore the folder's existence as well. As you alluded to earlier, we can always narrow it down differently later if it becomes important to massage.